build/config.cmake

```diff
@@ -36,6 +36,10 @@ IF (${WALRUS_OUTPUT} STREQUAL "shared_lib" AND ${WALRUS_HOST} STREQUAL "android"
     SET (WALRUS_LDFLAGS ${WALRUS_LDFLAGS} -shared)
 ENDIF()
 
+IF (WALRUS_EXTENDED_FEATURES)
+    SET (WALRUS_DEFINITIONS ${WALRUS_DEFINITIONS} -DENABLE_EXTENDED_FEATURES)
```

Comment:
Add build option `-DWALRUS_EXTENDED_FEATURES` to enable extended features (thread)